opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java

```diff
@@ -220,7 +220,7 @@ protected MLPredictionOutput getMLPredictionResult(FunctionName functionName,
 
     return (MLPredictionOutput) machineLearningClient
             .trainAndPredict(mlinput)
-            .actionGet(30, TimeUnit.SECONDS);
+            .actionGet(30L, TimeUnit.SECONDS);
```

Comment:
Is this change required?

Reply:
Nope, thanks. Removed both instances of the added `L`